src/client/Client.cc

```diff
@@ -10302,7 +10302,13 @@ int Client::ll_lookup(Inode *parent, const char *name, struct stat *attr,
   string dname(name);
   InodeRef in;
 
-  r = _lookup(parent, dname, CEPH_STAT_CAP_INODE_ALL, &in, perms);
+  if (dname == "..") {
+    r = _lookup(parent, dname, CEPH_MDS_OP_LOOKUPPARENT, &in, perms);
+  }
```

Comment:
nit: we usually do:
`
} else {
`
...all on one line

Reply:
@jtlayton Jeff, the work is in progress. I will push in the new fix. Thanks